onenote/endpoint.bal

```diff
@@ -0,0 +1,274 @@
+                                                         @display {label: "Section Name"} string sectionName) returns 
+                                                         Section|error {
+        string path = SECTION_GROUP_PATH + sectionGroupId + SECTIONS;
+        http:Response response = check self.httpClient->post(path, {displayName: sectionName});
+        map<json> handledResponse = check handleResponse(response);
+        return check handledResponse.cloneWithType(Section);
+    }
+
+    // Pages
+
+    # Lists pages of a section.
+    # 
+    # + sectionId - Section Id
+    # + oDataQuery - Optional. OData Query. Example: "$top=2&$count=true". 
+    # For details, refer https://docs.microsoft.com/en-us/graph/query-parameters#odata-system-query-options
+    # + return - Page[] if successful or else an error
+    @display {label: "List Pages"}
+    remote isolated function listPages(@display {label: "Section Id"} string sectionId, 
+                                       @display {label: "OData Query"} string? oDataQuery = ()) returns Page[]|error {
+        string path = SECTIONS_PATH + sectionId + PAGES;
+        if (oDataQuery is string) {
+            path = path + QUESTION_MARK + oDataQuery;
+        }
+        http:Response response = check self.httpClient->get(path);
+        return getPagesArray(response);
+    }
+
+    # Gets a page.
+    # 
+    # + pageId - Page Id
+    # + return - Page if successful or else an error
+    @display {label: "Get Page"}
+    remote isolated function getPage(@display {label: "Page Id"} string pageId) returns Page|error {
+        string path = PAGES_PATH + pageId;
+        http:Response response = check self.httpClient->get(path);
+        map<json> handledResponse = check handleResponse(response);
+        return check handledResponse.cloneWithType(Page);
+    }
+
+    # Creates a page.
+    # 
+    # + sectionId - Section Id
+    # + htmlContent - HTML Content
+    # + return - Page if page creation is successful or else an error
+    @display {label: "Create Page"}
+    remote isolated function createPage(@display {label: "Section Id"} string sectionId,
+                                        @display {label: "HTML Content"} string htmlContent) returns Page|error {
+        string path = SECTIONS_PATH + sectionId + PAGES;
+        http:Request request = new();
+        request.setTextPayload(htmlContent);
+        check request.setContentType("text/html");
+        http:Response response = check self.httpClient->post(path, request);
+        map<json> handledResponse = check handleResponse(response);
+        return check handledResponse.cloneWithType(Page);
+    }
+
+    # Deletes a page.
+    # 
+    # + pageId - Page Id
+    # + return - `true` if the page deletion is successful or else an error
+    @display {label: "Delete Page"}
+    remote isolated function deletePage(@display {label: "Page Id"} string pageId) returns boolean|error {
```

Comment:
Why do we need to return `true` for delete operation? We return `error?` in other connectors

Reply:
If we don't return anything for a successful operation, user won't have any idea if this operation was really succesful.

boolean will be easy for a developer when he is doing some other logic depending on the response of this remote operation.

Reply:
Done according to ur suggestion